draft-ietf-tls-esni.md

```diff
@@ -1761,7 +1782,11 @@ server name, such as ALPN preferences.
 ECH mitigates this attack by only negotiating TLS parameters from
 ClientHelloInner and authenticating all inputs to the ClientHelloInner
 (EncodedClientHelloInner and ClientHelloOuter) with the HPKE AEAD. See
-{{authenticating-outer}}. An earlier iteration of this specification only
+{{authenticating-outer}}. The decompression process in {{encoding-inner}}
+forbids "encrypted_client_hello" in OuterExtensions. This ensures the
+unauthenticated portion of ClientHelloOuter is not incorporated into
+ClientHelloInner.
+An earlier iteration of this specification only
 encrypted and authenticated the "server_name" extension, which left the overall
 ClientHello vulnerable to an analogue of this attack.
```

Comment:
Orthogonal: Do we want to add a note for the RFC editor to drop this text about "an earlier version of this specification"?

Reply:
No, I think this is helpful